ui/shared/components/panel/search/constants.js

```diff
@@ -472,13 +472,6 @@ export const IN_SILICO_FIELDS = [REQUIRE_SCORE_FIELD, ...PREDICTOR_FIELDS.filter
     const labelHelp = (
       <div>
         {`Enter a numeric cutoff for ${label}`}
-        {dangerThreshold && (
```

Comment:
I believe this is dead code after the recent changes... is that correct?

Reply:
yeah, but actually we should be replacing this with updated logic from the config thresholds, not just removing it. I would leave it for now and make a ticket to properly update it (or you can give that a go if you want)